02.calendar/calendar.rb

```diff
@@ -0,0 +1,58 @@
+#! /usr/bin/env ruby
+require "date"
+require "optparse"
+
+# ライブラリDate 本日を取得
+today = Date.today
+
+# ライブラリoptparse コマンドラインのオプション
+params = ARGV.getopts("m:y:")
+input_month = params["m"] ? params["m"] : today.mon
```

Comment:
このままだと input_month に文字列が入ったり数値が入ったり、データ型に一貫性がなくなるので、このタイミングで数値に揃えておきましょう。

```suggestion
input_month = params["m"] ? params["m"].to_i : today.mon
```